src/app/digital/actions/ConstantNumberChangeAction.ts

```diff
@@ -0,0 +1,36 @@
+import {Action} from "core/actions/Action";
+import {ConstantNumber} from "digital/models/ioobjects";
+
+/**
+ * An action to change the input for a ConstantNumber
+ */
+export class ConstantNumberChangeAction implements Action {
+
+    // the ConstantNumber component
+    private constantNumber: ConstantNumber;
+
+    // the input values
+    private inputNumber: number;
+    private inputNumberPrev: number;
```

Comment:
let's stick with naming scheme we use for other actions which is `initialNum` and `targetNum`